src/mate/academy/myArrayList/MyArrayList.java

```diff
@@ -0,0 +1,195 @@
+package mate.academy.myArrayList;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+public class MyArrayList<T> implements List<T> {
+
+    private Object[] array;
+    private int size;
+
+    public MyArrayList() {
+        array = new Object[10];
+    }
+
+    public MyArrayList(int capacity) {
+        if (capacity < 0)
+            throw new IllegalArgumentException("You was inputed incorect data.");
+        else if (capacity == 0)
+            array = new Object[capacity];
+    }
+
+    @Override
+    public int size() {
+        return size;
+    }
+
+    @Override
+    public boolean isEmpty() {
+        return size == 0;
+    }
+
+    @Override
+    public boolean contains(Object value) {
+        if (value == null) {
```

Comment:
this is not correct your array can contain multiple null elements and your array does not distinguish between added by user null elements and empty array indexes, do not allow add null elements

Reply:
@Siril-O i don't understand you, what i should change in this method?